.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_BASE_URL=http://localhost:8080/
```

Comment:
Add .env file to .gitignore file, and remove chache to delete the tracking of this file.

Reply:
aw sh"·$"·t